spec/mailers/order_mailer_spec.rb

```diff
@@ -255,4 +255,14 @@
       end
     end
   end
+
+  describe '.invoice' do
+    before do
+      @notification = OrderMailer.invoice(order.reload.id)
+    end
```

Comment:
why reload?

Reply:
Also we should not use instance variables in before blocks, it's [better](http://www.betterspecs.org/#let) to use `let`:
```
describe '.invoice' do
  let(:notification) { OrderMailer.invoice(order.id) }
  
  it 'delivers to all users in the buyer organization' do
    expect(notification).to deliver_to(buyer_user.email)
  end
end

Reply:
Good call. I was on copy-past auto-pilot from the gnarly specs above it. Will clean up.